vmhooks/baseOps.md

```diff
@@ -178,6 +205,12 @@ module BASEOPS
          <callValue> 0 </callValue>
          <esdtTransfers> .List </esdtTransfers>
 
+    // TODO check ESDT payment
+    rule <instrs> hostCall("env", "checkNoPayment", [ .ValTypes ] -> [ .ValTypes ]) 
+               => #throwException(ExecutionFailed, "function does not accept EGLD payment") ... </instrs>
```

Comment:
Could you move `</instrs>` or `... </instrs>` to the next line and align it with the open tag?